service-tests/hhvm.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'hhvm', title: 'hhvm status' });
+module.exports = t;
+
+t.create('get symfony status (default branch - not tested)')
+    .get('/symfony/symfony.json')
+    .expectJSONTypes(Joi.object().keys({ name: 'hhvm', value: 'not tested' }));
+
+t.create('get yii status (default branch - partially tested)')
+    .get('/yiisoft/yii.json')
+    .expectJSON({ name: 'hhvm', value: 'partially tested' });
+
+t.create('gets yii 1.1.19 status')
+    .get('/yiisoft/yii/1.1.19.json')
+    .expectJSON({ name: 'hhvm', value: 'partially tested' });
+
+t.create('gets yii 1.1.666 status (invalid branch)')
+    .get('/yiisoft/yii/1.1.666.json')
+    .expectJSON({ name: 'hhvm', value: 'invalid' });
+
+t.create('invalid package name')
+    .get('/frodo/is-not-a-package.json')
+    .expectJSON({ name: 'hhvm', value: 'invalid' });
```

Comment:
The 'invalid' label should be used when there is a problem generating the response (e.g: the json response could not be parsed). There is a handy fixture you can import using `const { invalidJSON } = require('./helpers/response-fixtures');` which allows you to mock an invalid response like this https://github.com/badges/shields/blob/75503ec5144b349ab42c4877b774d3acac8157f3/service-tests/cdnjs.js#L28-L34

In the case where the package or branch is not found, we should return a 'not found' rather than 'invalid'. It looks like where the repo is not found, php-eye returns a 404: https://php-eye.com/api/v1/package/frodo/not-a-package.json This is the most common case so we have a handler for it `checkErrorResponse` which can be used like this https://github.com/badges/shields/blob/75503ec5144b349ab42c4877b774d3acac8157f3/server.js#L6157-L6160 In this case, it would also be helpful to have distinct 'repo not found'/'package not found' and 'branch not found' messages.

`checkErrorResponse` will also handle the 'inaccessible' case, so you can remove the `if (err != null) ...`  block and replace with a block like the one above.